install_scripts/install_sindri.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Install sindri and related software
+
+if [[ $# -ne 1 ]]; then
+    echo "Pass the sensor number"
+    exit 1
+fi
+
+name="hamma"$1  # This will be used for the server push location
+
+# We need to modify a file (for now). Set up the variables for that
+LEKTOR_FILE="/home/pi/dev/sindri/src/sindri/website/mjolnir-website/mjolnir-website.lektorproject"
+LEKTOR_KEY="target = rsync://pi@hamma.dev/var/www/hamma.dev/public_html/"
+
+INSTALL_PATH="/home/pi/dev/"  # Trailing slash, please
+VENV_NAME="sindrienv"
+
+
+# Now, we're going to set up the environment
+
+python3 -m venv $INSTALL_PATH$VENV_NAME
+
+cp $INSTALL_PATH$VENV_NAME"/bin/activate" "/home/pi/$VENV_NAME"
+
+source "/home/pi/$VENV_NAME"
+
+# Note: not sure if this is needed
+pip install --upgrade setuptools wheel
```

Comment:
Instead of using a new env here, you might just want to share an env with Brokkr, since Sindri 0.3.x requires Brokkr 0.4.x in order to retrieve the various bits of config it needs (though I _could_ factor that out to a separate package, as I originally intended, it would be a fair chunk of work for relatively modest practical gain).

Reply:
I think ultimately we won't need this at all, as sindri will go server side. Since we had two envs in testing/development, I just kept it for the scripts. Longer term, we probably could just install as part of the `ltgenv` environment. But, I don't want to hold up the PR for that.